### ISSUE TITLE
RuntimeWarning in pca.py

### ISSUE THREAD

**Issue by stepenik** [povilasb/face-recognition#2]
Hi,
When you start program in the pca.py I see error RuntimeWarning

RuntimeWarning: invalid value encountered in sqrt
  S = np.sqrt(e[::-1]) # reverse since eigenvalues are in increasing order

https://github.com/povilasb/eigenfaces/blob/11f93098eabd4f16d21f40eb5173628eec9d6bef/pca.py#L23

Do you know how to fix that. This is not critical bug :)

**Comment by povilasb** [comment]
Not really, I forked eigenfaces from https://github.com/antonyross/eigenfaces.
I'll take a look at it anyway :)

**Comment by stepenik** [comment]
Ok thank you :)

**Comment by stepenik** [comment]
Do you looked this file :) .... Any news?

**Comment by povilasb** [comment]
Yes, the problem lays here https://github.com/povilasb/face-recognition/blob/master/facerecognition/eigenfaces/pca.py#L23

Basically I try to do square root for `e` vector:
```
 S = np.sqrt(e[::-1]) 
```
but one of `e` values is negative.

The easiest wayt to fix this would be to `abs` the vector `e` values.
But I'm not sure if that would make sense regarding PCA algorihtm.
I don't really understand it yet.

**Comment by stepenik** [comment]
Yes I am now test your case. If I put ` S = np.sqrt(np.abs(e[::-1])) ` in eigenface min_distance is higher.
For example if you dont put np.abs() min_distance is between 5600-6100. If you put np.abs() min_distance is between 6100-6500.
And I think it is better without `np.abs()`
